source/libs/api.ts

```diff
@@ -101,18 +110,35 @@ export const v3 = mem(async (
 	const textContent = await response.text();
 
 	// The response might just be a 200 or 404, it's the REST equivalent of `boolean`
-	const apiResponse: JsonObject = textContent.length > 0 ? JSON.parse(textContent) : {};
+	const apiResponse: JsonObject = json ? JSON.parse(textContent.length > 0 ? textContent : '') : {textContent};
```

Comment:
I think that doesn't fix the issue here, because it still tries to do `JSON.parse('')` 👀 

Maybe something like this would be enough:
```suggestion
	const apiResponse: JsonObject = (json && textContent.length > 0) ? JSON.parse(textContent) : {textContent};
```